openwisp_notifications/api/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
+from openwisp_notifications.api import views
+
+app_name = 'openwisp_notifications'
+
+urlpatterns = [
+    url(
+        r'^openwisp_notifications/notifications/$',
+        views.list_notifications,
+        name='api_list_notifications',
+    ),
+    url(
+        r'^openwisp_notifications/notifications/(?P<notification_id>[0-9A-Fa-f-]+)/$',
+        views.read_notification,
+        name='api_read_notifications',
```

Comment:
same here `read_notification`. since we are getting a single notification, keep away the `s`